fw/t/unit/test_http1_parser.c

```diff
@@ -3125,6 +3125,7 @@ TEST(http1_parser, fuzzer)
 		}
 	}
 end:
+	tfw_frames_chunks_free();
```

Comment:
I didn't get why do we need to free the chunks here? It seems all the chunks users, like `FOR_REQ()`, just rely on the fact that any following test frees the chunks in `tfw_init_frames()`, then why this doesn't work for this test case? Shouldn't we move the chunks freeing to the end of `TEST_SUITE()`?

Reply:
Yep, seems excessive.